build.gradle

```diff
@@ -0,0 +1,98 @@
+plugins {
+    id 'java-gradle-plugin'
+    id 'groovy'
+    id "com.gradle.plugin-publish" version "0.10.0"
+    id "com.github.hierynomus.license" version "0.14.0"
+}
+
+group = 'org.moditect'
+version = '1.0.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
```

Comment:
Defining `jcenter()` is enough as it contains all entries from `mavenCentral()`.